Userland/Utilities/nc.cpp

```diff
@@ -211,8 +233,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         }
 
         if (!fd_closed && FD_ISSET(fd, &readfds)) {
-            Array<u8, 1024> buffer;
-            Bytes buffer_span = buffer.span();
+            auto buffer = ByteBuffer::create_uninitialized(receive_buffer_size);
```

Comment:
```suggestion
            auto buffer = TRY(ByteBuffer::create_uninitialized(receive_buffer_size));
```
This will make `buffer` a `ByteBuffer`, rather than an `ErrorOr<ByteBuffer>`. If there is an error creating the buffer (e.g. no memory), this will then gracefully fail and return the error.